generate/Cargo.lock

```diff
@@ -1,1732 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
Why did you decide to remove the Cargo.lock? Usually it is ok to remove lock for libs but to commit it for bins, and here it is a binary crate.

Reply:
Because there is a single lock for a workspace

Reply:
Oh, ok